addon/components/listbox/-options.hbs

```diff
@@ -0,0 +1,27 @@
+{{#if @isOpen}}
+  <ul
```

Comment:
It would be good to support changing the tag name used here, as we can with the other components by setting an `@tagName` argument

Reply:
https://github.com/GavinJoyce/ember-headlessui/pull/76/commits/2489d1156b5cc1016a184acf1a1e92202c0f9a37 allows custom tag names for this component.

I'm using the `@as` property to specify a tag name, as this is consistent with the React API and is also a pattern that `<Dialog>`  follows.

Reply:
Huh, I didn't realize we were _already_ inconsistent about whether to use `@tagName` or `@as`

I like `@as` better anyway, so sticking with that makes sense to me! I'll update the other components to follow that pattern